### PR TITLE
Update fileupload.json

### PR DESCRIPTION
**Pull request by KrunchMuffin**
added boolean strict to syntax

### CODE REVIEW

File: data/en/fileupload.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fileUpload",
   "type": "function",
-  "syntax": "fileUpload(string destination,[string fileField,[string accept,[string nameConflict]]])",
+  "syntax": "fileUpload(string destination,[string fileField,[string accept,[string nameConflict,[boolean strict]]]])",
```
Comment (shaedrich) [inline data/en/fileupload.json]: We do not use data type in `syntax`. Could be removed.

**Comment by shaedrich** [comment]
Why did you close your pull request? This was right. You just had to remove the data types which could have been done in another pull request as well so you wouldn't necessarily have to care about that.

**Comment by KrunchMuffin** [comment]
My bad.

**Comment by shaedrich** [comment]
Can you re-open it?

File: data/en/fileupload.json (hunk 1 above)
Comment (shaedrich) [reply data/en/fileupload.json]: @KrunchMuffin contributed at 10th November for the last time until now, so I don't know, if he will come back...

**Comment by KrunchMuffin** [comment]
I didn't know what u were talking about since I didn't add the datatypes
that were existing. All I did was add strict attribute and followed what
was already there.


On Nov 21, 2017 at 11:07 AM, <Sebastian Hädrich <notifications@github.com>>
wrote:

*@shaedrich* commented on this pull request.
------------------------------

In data/en/fileupload.json
<https://github.com/foundeo/cfdocs/pull/706#discussion_r152321879>:

> @@ -1,7 +1,7 @@
 {
   "name": "fileUpload",
   "type": "function",
-  "syntax": "fileUpload(string destination,[string fileField,[string
accept,[string nameConflict]]])",
+  "syntax": "fileUpload(string destination,[string fileField,[string
accept,[string nameConflict,[boolean strict]]]])",

@KrunchMuffin <https://github.com/krunchmuffin> contributed at 10th
November for the last time until now, so I don't know, if he will come
back...

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/foundeo/cfdocs/pull/706#discussion_r152321879>, or mute
the thread

**Comment by shaedrich** [comment]
No offense. I just mentioned that it was wrong before you edited the file. Because you edited it, this would be a chance to make it right. No more, no less.

**Comment by KrunchMuffin** [comment]
No offense. Just didn't understand cause the types were already there.


On Nov 21, 2017 at 12:35 PM, <Sebastian Hädrich <notifications@github.com>>
wrote:

No offense. I just mentioned that it was wrong before you edited the file.
Because you edited it, this would be a chance to make it right. No more, no
less.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/foundeo/cfdocs/pull/706#issuecomment-346102863>, or mute
the thread

**Comment by pfreitag** [comment]
No worries Derek - we are just trying to remove inconsistencies in the docs so as changes come in to a doc if someone sees something that is not consistent they might point it out.

I realize you were just adding on to what was there... what was already there is where the problem was.

**Comment by pfreitag** [comment]
Merging this PR, I will take care of the old syntax fix thanks  @KrunchMuffin

**Pull request merged**